### PR TITLE
Created Fluo 1.1.0 release notes

### PR DESCRIPTION
**Pull request by keith-turner**

### CODE REVIEW

File: _posts/release/2017-06-02-fluo-1.1.0-incubating.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+[bin-release]: https://www.apache.org/dyn/closer.lua/incubator/fluo/fluo/1.1.0-incubating/fluo-1.1.0-incubating-bin.tar.gz
+[bin-asc]: https://www.apache.org/dist/incubator/fluo/fluo/1.1.0-incubating/fluo-1.1.0-incubating-bin.tar.gz.asc
+[bin-md5]: https://www.apache.org/dist/incubator/fluo/fluo/1.1.0-incubating/fluo-1.1.0-incubating-bin.tar.gz.md5
+[bin-sha]: https://www.apache.org/dist/incubator/fluo/fluo/1.1.0-incubating/fluo-1.1.0-incubating-bin.tar.gz.sha
+[src-release]: https://www.apache.org/dyn/closer.lua/incubator/fluo/fluo/1.1.0-incubating/fluo-1.1.0-incubating-source-release.tar.gz
+[src-asc]: https://www.apache.org/dist/incubator/fluo/fluo/1.1.0-incubating/fluo-1.1.0-incubating-source-release.tar.gz.asc
+[src-md5]: https://www.apache.org/dist/incubator/fluo/fluo/1.1.0-incubating/fluo-1.1.0-incubating-source-release.tar.gz.md5
+[src-sha]: https://www.apache.org/dist/incubator/fluo/fluo/1.1.0-incubating/fluo-1.1.0-incubating-source-release.tar.gz.sha
+[javadocs]: {{ site.fluo_api_base }}/1.1.0-incubating/
+[docs]: /docs/fluo/1.0.0-incubating/
```
Comment (mikewalch) [inline _posts/release/2017-06-02-fluo-1.1.0-incubating.md]: should be `1.1.0-incubating`

File: _posts/release/2017-06-02-fluo-1.1.0-incubating.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
+---
+title: Apache Fluo 1.1.0-incubating released
+date: 2017-06-02 15:00:00 +0000
```
Comment (mikewalch) [inline _posts/release/2017-06-02-fluo-1.1.0-incubating.md]: This date should probably be 2017-06-12

**Review by mikewalch — APPROVED** [review]

File: _posts/release/2017-06-02-fluo-1.1.0-incubating.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+groups and each group scans a subset of the table for notifications.  Every worker in a group scans
+the groups entire subset of a table  looking for notifications that hash to it. The new strategy results
+in O(N*G) work where N is the number of notifications and G is the group size.  By default the group
+size is 7, but this is configurable.  The group size may need to be increased if portion of a table
+is popular and assigned to one group that can not processes it.
+
+To compare the old and new ways assume we have 10<sup>9</sup> notifications and 100 workers.  The
+old method would have scanned 10<sup>11</sup> entries to to find all notifications.  Assuming a group
+size of 7, the new strategy scans 7 * 10<sup>9</sup> entries to find all notifications.  A
+nice feature of the new strategy is that the amount of scanning is independant of the number of workers.
```
Comment (cjmctague) [inline _posts/release/2017-06-02-fluo-1.1.0-incubating.md]: independant should be independent

File: _posts/release/2017-06-02-fluo-1.1.0-incubating.md [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+### Improved Bytes
+
+Fluo's API has an immutable wrapper for a byte array.  Multiple improvements were made to this byte
+wrapper.
+
+  * `startsWith(Bytes)` and `endsWith(Bytes)` methods were added in [#823]
+  * A `copyTo(byte[])` method was added for [#827]
+  * Internal performance improvements were made in [#826] and [#799]
+
+### Improved Spark intergration
```
Comment (cjmctague) [inline _posts/release/2017-06-02-fluo-1.1.0-incubating.md]: intergration should be Integration

**Review by cjmctague — CHANGES_REQUESTED** [review]

File: _posts/release/2017-06-02-fluo-1.1.0-incubating.md [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+  }
+
+  /**
+   * Compute the change in a documents URLs and propagate those to other documents.
+   */
+  private static class ContentObserver implements StringObserver {
+    @Override
+    public void process(TransactionBase tx, String myUrl, Column col) throws Exception {
+
+      // Retrieve the new and current docuemnt content
```
Comment (cjmctague) [inline _posts/release/2017-06-02-fluo-1.1.0-incubating.md]: docuemnt should be document

**Review by cjmctague — CHANGES_REQUESTED** [review]

**Review by cjmctague — APPROVED** [review]

**Pull request merged**